cpp/ql/test/library-tests/syntax-zoo/dataflow-ir-consistency.expected

```diff
@@ -564,11 +564,17 @@ uniqueNodeToString
 | break_labels.c:2:11:2:11 | x | Node should have one toString but has 2. |
 | break_labels.c:2:11:2:11 | x | Node should have one toString but has 2. |
 | break_labels.c:4:9:4:9 | i | Node should have one toString but has 2. |
+| break_labels.c:4:9:4:9 | i indirection | Node should have one toString but has 2. |
```

Comment:
Should I be worried about the changes to this file?

Reply:
Or is this basically a consequence of the direct node having the same problem and the strings being constructed differently now?

Reply:
Yeah, I think so.

Reply:
You think I should be worried, or you think it's a consequence of? 😄